query-engine/driver-adapters/executor/src/qe.ts

```diff
@@ -0,0 +1,71 @@
+import type {
```

Comment:
I haven't changed this file much, only to pass the engine type as an argument rather than reading it from the env. Because I need to create two engines for the two different engine types in order to compare them in the benchmarks.